README.md

```diff
@@ -1 +1,33 @@
-# lab-13-user-adventure
\ No newline at end of file
+# lab-13-user-adventure
```

Comment:
great work with planning, your wireframe, etc.  Your planning is very detailed, and you're doing a wonderful job :)